Formula/g/gauth.rb

```diff
@@ -0,0 +1,17 @@
+class Gauth < Formula
+  desc "Google Authenticator in your terminal"
+  homepage "https://github.com/pcarrier/gauth"
+  url "https://github.com/pcarrier/gauth/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "5c98287f5c209b9f02ec62ede4abd4117aa3ca738fbcb4153a6ec1e966f492a8"
+  license "ISC"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match(/Examples:/, shell_output(bin/"gauth"))
```

Comment:
Is it possible to make the test a bit more specific to the gauth functionality?

Reply:
Kinda hard given the tokens change every 30s.

Reply:
Tests have their own home directory

Reply:
Thanks! Added a basic test.

Reply:
Thanks!